platform/entanglement/src/actions/CopyActionWizard.js

```diff
@@ -0,0 +1,87 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2017, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+define(
+    function () {
+        function CopyActionWizard(domainObject, parent, locationValidator) {

```

Comment:
This wizard is almost 100% identical to the CreateWizard. The two have different purposes (in that they're used for different purposes, but both create a model), but finding a way to merge them together would get rid of that extra Spec file. 

Yeah, I agree. Let's see if we can merge them. In fact, the [`PropertiesDialog`](https://github.com/nasa/openmct/blob/master/platform/commonUI/edit/src/actions/PropertiesDialog.js) is also basically the same except for the location selector. Perhaps we could use the `PropertiesDialog` as a base class and extend it with a `NewObjectDialog` that includes the location control. Then we can use the `NewObjectDialog` from the CreateAction and the `CopyAction`. Thoughts?

CreateWizard and CopyActionWizard also have a `createModel` function on them. It basically does the same thing as `updateModel` on `PropertiesDialog` so I'd be inclined to get rid of `createModel` altogether and just externalize the instantiation of the new model object and pass it into `updateModel`.